### ISSUE TITLE
Is this project actively maintained? 

### ISSUE THREAD

**Issue by nunocoracao** [nodegit/nodegit#2002]
Just wanted to know if the project is actively maintained or now. Last release seems quite old.

**Comment by sneakycrow** [comment]
Not a maintainer, just a consumer. I also looked into this after seeing the latest GitHub release specifically.

The formal GitHub Releases do seem to be quite old, but if you look at the [tags](https://github.com/nodegit/nodegit/tags) you can see more current versions are being tagged. And a look at that [npm repository](https://www.npmjs.com/package/nodegit) shows a publish as recently as 16 days ago.

Additionally, the [commit history](https://github.com/nodegit/nodegit/commits/master/) does seem to be active as well.

I would say it is maintained, albeit minus keeping the releases on GH up-to-date. Maybe the maintainer just aren't using GH Releases anymore.

**Comment by nunocoracao** [comment]
Yup, same thing here. However, I was trying an npm install and it didn't work hence why I was checking. I was able to run it when compiling the repo locally and importing it locally into my project though...

**Comment by abhishiv** [comment]
Figma maintains prebuilt binaries for nodegit with non deprecated node versions, but unfortunately they don't have one for mac os so local development using that sucks

https://github.com/nodegit/nodegit/issues/1980#issuecomment-2241021870
